pkg/synthetictests/historicaldata/next_best_guess.go

```diff
@@ -49,6 +49,7 @@ var nextBestGuessers = []NextBestKey{
 
 	combine(ForTopology("single"), OnSDN),
 	combine(ForTopology("single"), OnSDN, PreviousReleaseUpgrade),
+	combine(ForTopology("single"), OnSDN, PreviousMicroReleaseUpgrade),
```

Comment:
does `combined(forTopology, onsdn, previousreleaseupgrade, microreleaseupgrade)` work?

Reply:
Yup! Thank you! Updated :)